src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -617,6 +618,48 @@ def handler(args):
     context._cli_command(name, handler=handler, argument_loader=generic_wait_arguments_loader, **kwargs)  # pylint: disable=protected-access
 
 
+def _cli_generic_show_command(context, name, getter_op, **kwargs):
+
+    if not isinstance(getter_op, string_types):
+        raise ValueError("Getter operation must be a string. Got '{}'".format(type(getter_op)))
+
+    factory = _get_client_factory(name, kwargs)
+
+    def generic_show_arguments_loader():
+
+        getter_args = dict(extract_args_from_signature(context.get_op_handler(getter_op),
+                                                       excluded_params=EXCLUDED_PARAMS))
+        cmd_args = getter_args.copy()
+        cmd_args['cmd'] = CLICommandArgument('cmd', arg_type=ignore_type)
+        return [(k, v) for k, v in cmd_args.items()]
```

Comment:
Is this the same code as generic wait/update?  If so we could probably extract these to a common implementation for maintainability.